src/components/common/TmBalance.vue

```diff
@@ -16,6 +16,7 @@
         <h3>Rewards</h3>
         <h2>{{ rewards }}</h2>
         <TmBtn
+          v-show="rewards > 0"
```

Comment:
Why not disabling it? Hiding it does also hide the functionality.

Reply:
Good question

It's a link, not a button, and perhaps 'dead text' could be confusing? I could do that and add a tool tip perhaps? Explaining why it's not 

Then again hiding it initially is maybe not a bad thing? As a user gets rewards it becomes intuitive why it's not there at the beginning (or again not present in the future)

I'll experiment, and post a gif.

Reply:
@jbibla Do you have thoughts on this? I'm hesitant to 'disable' an anchor text. Not quite sure what it should look like. A more dull colour with no cursor pointer?

Reply:
Ah! Sorry I was mistaken. I thought this is the button on the Actionmodal. I think this is fine.